e2e/docker-compose.yml

```diff
@@ -27,7 +28,12 @@ services:
     depends_on:
       - panacea
     command: >
-      /e2e/wait-for panacea:9090 --timeout=600 -- datavald
+      sh -c "
+              echo 'wait 60 sec...' &&
+              sleep 60 &&
+              echo 'end wait'
+              /e2e/wait-for panacea:9090 --timeout=600 -- datavald
```

Comment:
If you do tx before the block in the chain is created, it will not work properly, so you have to wait for the block to be created.
I've been guiding Incheol like this, but I have to think about if there's a better way.
As i left a comment below, there is also a way to register by execution the command.

Reply:
Good to sleep 60s, IMO.

BTW, as I know, we can execute multi-line commands as below (without `sh -c "..."`):
```
run: |
  cmd1
  cmd2
```
Example: https://github.com/medibloc/panacea-js/blob/1eb19e321314c7c72c5b4a7cf5d87d0cc2f9777f/.github/workflows/ci.yml#L36

But, I'm not sure. We need to test it.

Reply:
Changed in [2366c6c](https://github.com/medibloc/panacea-data-market-validator/pull/68/commits/2366c6c0711c3acc8e2dd25bd13168ff633a39b6), but still use `sh -c "..."`